tests/tests_unit/test_api_client.py

```diff
@@ -1147,6 +1147,7 @@ class TestHelpers:
             ("PUT", "https://localhost:8000.com/api/v1/projects/blabla/assets", True),
             ("PATCH", "https://localhost:8000.com/api/v1/projects/blabla/patchy", True),
             ("GET", "https://another-cluster.cognitedata.com/login/status", True),
+            ("POST", "https://another-cluster.cognitedata.com/api/v1/projects/hey/raw", True),
```

Comment:
should be longer?
`https://api.cognitedata.com/api/v1/projects/{project}/raw/dbs/{dbName}/tables/{tableName}/rows`
https://docs.cognite.com/api/v1/#tag/Raw/operation/postRows

Reply:
Oh yeah. Forgot the en and table was specified in the path. Good catch!